src/RecognitionClient.cpp

```diff
@@ -264,6 +267,31 @@ RecognitionClient::convertTopic(const std::string &topicName) {
     return topicIter->second;
 }
 
+std::unique_ptr<GrammarResource>
+RecognitionClient::buildGrammarResource(const Grammar &grammar) {
+    std::unique_ptr<GrammarResource> resource(
```

Comment:
it's better to use the `std::make_unique<>` to create a unique pointer, but as i wrote in my previous comment maybe it's a good idea to change the return type of the function to a plain pointer

Reply:
Changed